etc/kayobe/environments/ci-builder/stackhpc-ci.yml

```diff
@@ -23,7 +23,6 @@ kolla_enable_mariabackup: true
 kolla_enable_neutron_sriov: true
 kolla_enable_octavia: true
 kolla_enable_opensearch: true
-kolla_enable_ovn: true
```

Comment:
Please remove this from the other files I mentioned in the original PR

Reply:
Done.